### PR TITLE
Feat/database structure

### PR DESCRIPTION
**Pull request by akash-codemonk**

### CODE REVIEW

**Comment by hetunandu** [comment]
LGTM.
Requesting @riodeuno review for entity explorer related changes
and @mohanarpit for further db related testing

File: app/client/craco.common.config.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ module.exports = {
     {
       plugin: "prismjs",
       options: {
-        languages: ["javascript"],
+        languages: ["javascript", "sql"],
```
Comment (riodeuno) [inline app/client/craco.common.config.js:19]: I am looking to the reference for this. As in, the point in the PR code where we're using SQL syntax highlighting using prismjs.

File: app/client/src/components/editorComponents/HighlightedCode/index.tsx [hunk 2]
```diff
@@ -31,6 +31,7 @@ const StyledCode = styled.div<{ skin: Skin }>`
 /* When adding an entry please make sure to include it in the craco.common.config.js as well */
 export enum SYNTAX_HIGHLIGHTING_SUPPORTED_LANGUAGES {
   JAVASCRIPT = "language-javascript",
+  SQL = "language-sql",
```
Comment (riodeuno) [inline app/client/src/components/editorComponents/HighlightedCode/index.tsx:34]: Same as above, I cannot find any references to this. Please point me towards the references.

File: app/client/craco.common.config.js (hunk 1 above)
Comment (akash-codemonk) [reply app/client/craco.common.config.js]: Was showing the queries in highlighted form later it was decided we are going to show that as of now so removed them.

Will remove this.

**Review by riodeuno — APPROVED** [review]

**Pull request merged**